hydroshare/settings.py

```diff
@@ -918,6 +918,15 @@ def _(s): return s # noqa
         'django.contrib.auth.hashers.MD5PasswordHasher',
     ]
 
+    DATABASES['default'] = {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'PASSWORD': 'postgres',
+        'HOST': 'postgis',
+        'PORT': '5432',
+    }
```

Comment:
What you have here is good, it will solve the [DB drop issue for pgbouncer during tests](https://stackoverflow.com/questions/8047742/django-cant-drop-the-test-db-via-pgbouncer).

But there's a problem.
take a look at how the tests are run by the GH actions:
https://github.com/hydroshare/hydroshare/blob/2.15.9/run-tests-jenkins#L3-L5

So the problem is that 
`coverage run manage.py test -v 3 --with-xunit $1 --with-timer --exe`
has `'test' in sys.argv`
but
`pytest` does not have `'test' in sys.argv`
that means that pytest tests are going to still try to run through pgbouncer, I think.

So I would try changing the `if 'test' in sys.argv:` so that it also captures the pytest tests...maybe that will work

Reply:
Darn, [looks like that didn't work](https://github.com/hydroshare/hydroshare/actions/runs/12412160190/job/34651350517) hmmm....